docs/dom-testing-library/api-events.mdx

```diff
@@ -121,6 +121,8 @@ programmatically (such as `timeStamp`).
 const myEvent = createEvent.click(node, { button: 2 })
 fireEvent(node, myEvent)
 // myEvent.timeStamp can be accessed just like any other properties from myEvent
+// note: The access to the events created by `createEvent` is based on the native event API,
+// Therefore, native properties of HTMLEvent object (e.g. `timeStamp`) should be set using Object.defineProperty
```

Comment:
I'd probably also add that only some properties need to be changed using `Object.defineProperty` as there are no setters in the native implementation.

Reply:
@MatanBobi that's why I said "native" properties, but I'm honestly not exactly sure about the exact list

Reply:
Everything in [this page](https://developer.mozilla.org/en-US/docs/Web/API/Event) which has `Read only` on it can't be changed without using `Object.defineProperty` :)
I don't mean to add a list, maybe just some examples. As all properties are the native properties.

Reply:
@MatanBobi cool. adding